sites/public/src/lib/hooks.ts

```diff
@@ -142,6 +163,29 @@ export async function fetchClosedListings() {
 }
 
 let jurisdiction: Jurisdiction | null = null
+const bloomJurisdictions: Jurisdiction[] = []
+
+export async function fetchBloomJurisdictionsByName() {
+  if (bloomJurisdictions.length != 0) {
```

Comment:
I think we should consider caching these during the init stage of the application. This could prevent us from handling the race condition.

Reply:
I agree although my TS/react knowledge is not enough to know how to fetch the jurisdiction info from the API and effectively cache it on the client side outside of this approach -- cc @ludtkemorgan any thoughts?

Reply:
Resolving this to merge -- but def interested in learning more!